Pipfile

```diff
@@ -11,19 +11,23 @@ uWSGI = "*"
 WTForms = "==2.1"
 PyYAML = "*"
 pytz = "*"
-SQLAlchemy = "*"
+SQLAlchemy = "==1.2.17"
 bleach = "*"
 requests = "==2.19.1"
-arxiv-base = "==0.12.1"
-arxiv-submission-core = "==0.6.2rc3"
+arxiv-base = "==0.14.3"
 mypy_extensions = "*"
-arxiv-auth = "==0.2.6rc1"
+arxiv-auth = "==0.2.7rc1"
+mysqlclient = "==1.4.2"
+arxiv-submission-core = "==0.6.2rc5"
+mimesis = "==2.1.0"
 
 [dev-packages]
-pylint = "*"
 pydocstyle = "*"
-mypy = "*"
+mypy = "==0.670"
```

Comment:
Is there a particular reason for the version pin of mypy with `==` and not `>=`?

Reply:
Yeah, some of the AST-related deps run into conflicts, e.g. with pydocstyle, others. Better to be explicit